keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/TitusActiveServerGroup.kt

```diff
@@ -20,6 +20,55 @@ package com.netflix.spinnaker.keel.clouddriver.model
 import com.netflix.spinnaker.keel.api.Moniker
 import com.netflix.spinnaker.keel.core.api.Capacity
 
+data class TitusServerGroup(
+  val name: String,
+  val awsAccount: String,
+  val placement: Placement,
+  val region: String,
+  val image: TitusActiveServerGroupImage,
+  val iamProfile: String,
+  val entryPoint: String,
+  val targetGroups: Set<String>,
+  val loadBalancers: Set<String>,
+  val securityGroups: Set<String>,
+  val capacity: Capacity,
+  val cloudProvider: String,
+  val moniker: Moniker,
+  val env: Map<String, String>,
+  val containerAttributes: Map<String, String> = emptyMap(),
+  val migrationPolicy: MigrationPolicy,
+  val serviceJobProcesses: ServiceJobProcesses,
+  val constraints: Constraints,
+  val tags: Map<String, String>,
+  val resources: Resources,
+  val capacityGroup: String,
+  val disabled: Boolean
+)
+
+fun TitusServerGroup.toActive() =
+  TitusActiveServerGroup(
+    name,
```

Comment:
I'd suggest adding the property names here. It's very hard to correlate the positional arguments with the properties, especially when there's a bunch of string values after one another.

Reply:
Fixed in 306c3194